### PR TITLE
Issue 40 wrap up readme md for webgoat net milestone 0 1

### PR DESCRIPTION
**Pull request by tobyash86**
Creating pull request to track @annastuchlik changes

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Changelog
+
+## Version 0.1
+### Added
+
+### Fixed
+- Fixed varying number of featured products to be always to be always 4
```
Comment (tobyash86) [inline changelog.md:7]: to be always to be always - repetition

File: changelog.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### Removed
+- Removed raw SQL query from code base
```
Comment (tobyash86) [inline changelog.md:10]: Finally it was not done, we gave up on this. Please remove

**Review by tobyash86 — COMMENTED** [review]
Ania, please make corrections according to my comments.
Thanks.

**Comment by annastuchlik** [comment]
I've made the corrections.

**Pull request closed without merging**